src/components/Admin/Toolbar.jsx

```diff
@@ -135,6 +129,12 @@ const Toolbar = ({ input, setInput, tags, setObjects, objects, filters }) => {
           className="ml-5 text-hackathon-darkgray hover:opacity-70 duration-150"
         />
       </button>
+      <button className="ml-auto">
```

Comment:
remove the ml-auto, we should be able to determine the margin left without auto

Reply:
i found that making the margin to be 600px also works, can I do that?

Reply:
Find a way to do it without such a large margin